app/controllers/concerns/course/discussion/posts_concern.rb

```diff
@@ -1,7 +1,12 @@
-class Course::Discussion::PostsController < Course::ComponentController
```

Comment:
Modified abstract controller to a concern, because sometimes PostsController needs to inherit the basic controller of a specific component.